core/server/api/v2/posts.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../../models');
+
+module.exports = {
+    docName: 'posts',
+    browse: {
+        options: [
+            'include',
+            'filter',
+            'status',
+            'fields',
+            'formats',
+            'page',
+            'limit',
+            'order',
+            'debug'
```

Comment:
@kirrg001 where this debug field comes from and what's it's purpose here?

Reply:
I just copied it from v0.1. It's used in the model layer (if you search for options.debug)